src/os/bluestore/NVMEDevice.cc

```diff
@@ -138,8 +142,14 @@ class SharedDriverQueueData {
         flush_lock("NVMEDevice::flush_lock"),
         flush_waiters(0),
         completed_op_seq(0), queue_op_seq(0) {
+    struct spdk_nvme_io_qpair_opts opts = {};
+    spdk_nvme_ctrlr_get_default_io_qpair_opts(ctrlr, &opts, sizeof(opts));
```

Comment:
@optimistyzy , please correct me: 
this function is used to get io_queue_size(default queue pair iodepth), so why not just use this default value as max_queue_depth?